### DISCUSSION TITLE
Debian - Trying to uninstall but FreeTube not showing in dkpg or Software Centre.

### DISCUSSION THREAD

**Discussion by ChugAug** [FreeTubeApp/FreeTube#5206] [Q&A]
I'm trying to uninstall FreeTube. It opens just fine, shows up in the applications list on gnome and runs when "freetube" is entered in terminal. However, when trying to find it in the Software app or remove it in Terminal it doesn't appear. Typing sudo apt-get remove freetube shows the following: 

[No apt package "freetube", but there is a snap with that name. Try "snap install freetube" E: Unable to locate package freetube.]

I've enjoyed my time with Freetube but I don't watch youtube anymore so I'd like to remove it. Any suggestions?

Debian 12
Freetube version 0.20.0

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @ChugAug how did you install FT? Did you download it straight from our site?

**Reply by ChugAug** [reply]
Yes, I downloaded a .deb file of an earlier version. Either 0.16.0 or 0.19.0 I think.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
This worked for me

1. `sudo apt list --installed`
2. search for freetube just by using your eyes for me it was just listed as `freetube` but maybe its named a bit different on your system
3. `sudo apt purge INSERT_APPLICATION_NAME`

**Reply by ChugAug** [reply]
Just tried it. Freetube still isn't there. Did a search for freetube, nothing.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Lets try the following

1. Go to your applications
2. right click on FreeTube
3. click on `App Details`
4. it should correctly open up the `Software centre`
5. now you can remove it from there

**Reply by ChugAug** [reply]
When software centre opened it said at the top  "Sorry! There are no details for that application."

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Thats very weird. 
Maybe this is related https://askubuntu.com/questions/1321026/no-application-found-in-ubuntu-software-center-20-04 or this https://ubuntuforums.org/showthread.php?t=2453818

Maybe you did install it as a snap or flatpak version?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
try
`list snap`
`flatpak list`

search for freetube in there

**Reply by ChugAug** [reply]
There's 2 apps called Freedesktop. Would that be it? I read the first link you sent and I do recall seeing a file called freetube.desktop. What should I do with it?

Tried the second thing, the grep command. Here's what I got:
Name               Version          Rev    Tracking       Publisher         Notes
bare               1.0              5      latest/stable  canonical✓        base
core               16-2.61.2        16928  latest/stable  canonical✓        core
core22             20240408         1380   latest/stable  canonical✓        base
freetube           0.20.0           45     latest/stable  sameersharma2006  -
gnome-42-2204      0+git.510a601    176    latest/stable  canonical✓        -
gtk-common-themes  0.1-81-g442e511  1535   latest/stable  canonical✓        -

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
of what of the above listed commands is this the output for?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
I dont think its `Freedesktop` but the output of your grep command shows `freetube 0.20.0 45 latest/stable sameersharma2006`. Thats the one you want to remove.

**Reply by ChugAug** [reply]
> of what of the above listed commands is this the output for?

dpkg -l| grep freetube
snap list

Just tried flatpak uninstall freetube and it said: error: No installed refs found for ‘freetube’

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
from the output shown above freetube is installed as a snap. So you need to remove like this

`sudo snap remove package_name`

**Reply by ChugAug** [reply]
That removed it fully. Thanks so much. You're the best.
